kitsune/tidings/events.py

```diff
@@ -108,8 +104,30 @@ class Event(object):
     #: ``set(['color', 'flavor'])``
     filters = set()
 
-    def fire(self, exclude=None, delay=True):
-        """Notify everyone watching the event.
+    @classmethod
+    def fire_async(cls, instance=None, exclude=None):
```

Comment:
We still need some method that synchronously performs the work of the `fire` method, because the Celery task needs to call that, but we could call that something else instead, like `send_emails`, to make it less confusing?